src/lakefs_spec/spec.py

```diff
@@ -521,6 +521,15 @@ def ls(
                         }
                     )
                 elif isinstance(obj, ObjectInfo):
+                    # Skip over prefix-only matches, e.g.:
+                    # - foo/bar
+                    # -
```

Comment:
Does this empty bullet serve the explanation (in this case, I don't quite get it)?

Reply:
Thanks, that was a stray edit. I have improved the appearance in 83a9255.